x-pack/plugins/security_solution/public/hosts/store/model.ts

```diff
@@ -39,6 +40,7 @@ export interface Queries {
   [HostsTableType.uncommonProcesses]: BasicQueryPaginated;
   [HostsTableType.anomalies]: null | undefined;
   [HostsTableType.alerts]: BasicQueryPaginated;
+  [HostsTableType.risk]: null | undefined;
```

Comment:
i dont understand this type.. null or undefined? So its never anything?

Reply:
Yes, it doesn't get any value. I think it could be only `undefined`, but I lazily copied it from line 41. At least It is consistent. 😅